src/azure-cli/azure/cli/command_modules/synapse/manual/commands.py

```diff
@@ -583,6 +588,13 @@ def get_custom_sdk(custom_module, client_factory):
         g.custom_command('export', 'synapse_kusto_script_export')
         g.custom_wait_command('wait', 'synapse_kusto_script_show')
 
+    with self.command_group('synapse ad-only-auth', command_type=synapse_adonlyauthentications_sdk,
+                            custom_command_type=get_custom_sdk('adonlyauthentications', cf_synapse_client_azure_ad_only_authentications_factory),
+                            client_factory=cf_synapse_client_azure_ad_only_authentications_factory) as g:
+        g.custom_command('enable', 'synapse_enable_adonly_auth')
+        g.custom_command('disable', 'synapse_disable_adonly_auth')
```

Comment:
Why not use `az synapse ad-only-auth set --enable true/false`?

Reply:
Because sql related command are "sql server ad-only-auth enable/disable". I think it's properly to keep same.